src/GitVersionExe.Tests/GitVersionHelper.cs

```diff
@@ -2,26 +2,33 @@
 using System.Collections.Generic;
 using System.IO;
 using System.Text;
+
```

Comment:
Yes. I'm actually not entirely sure if it is my machine-level R# doing this or the local ("team shared") `sln.DotSettings` file. Either way, I agree that the local one should specify this explicitly.